app/js/app.js

```diff
@@ -133,4 +133,14 @@ alertaApp.config(['config', '$authProvider',
       clientId: config.client_id,
       authorizationEndpoint: config.keycloak_url+'/auth/realms/'+config.keycloak_realm+'/protocol/openid-connect/auth'
     });
+    $authProvider.oauth2({
+      name: 'pingfederate',
+      url: config.endpoint+'/auth/pingfederate',
+      redirectUri: window.location.origin+'/',
+      clientId: config.client_id,
+      authorizationEndpoint: config.pingfederate_url,
+      pfidpadapterid: 'kerberos',
+      scope: 'openid+profile+email',
+      response_type: 'code'
```

Comment:
Should `response_type` be `responseType`? If pingfederate works when this is misspelt maybe it's not actually needed.

Reply:
Looks like `code` is the default so this line can be removed. https://github.com/sahat/satellizer/blob/master/src/oauth2.ts#L59

Reply:
removed it & committed.